plugins/mumble_plugin_win32.h

```diff
@@ -18,9 +18,24 @@
 
 #include "mumble_plugin.h"
 
+#if _MSC_VER < 1800 && defined(M_IX86)
+typedef unsigned long long uintptr64_t;
+typedef unsigned long uintptr32_t;
+#else
+#include <stdint.h>
+typedef uint64_t uintptr64_t;
```

Comment:
Since this type is only going to be used for pointers in the memory space of the "module" and cannot be used as a normal pointer in the plugin I think calling it a "module pointer" is correct. If you have a type with narrow scope and use why not use it to your advantage ;) If you are really that fond of the uintptr32_t and uintptr64_t names feel free to use them but I (knowing uintptr_t) was quite confused about those seemingly new additions to the standard I didn't know about and couldn't google.